forge-gui/res/cardsfolder/upcoming/captain_america_first_avenger.txt

```diff
@@ -0,0 +1,9 @@
+Name:Captain America, First Avenger
+ManaCost:R W U
+Types:Legendary Creature Human Soldier Hero
+PT:4/4
+A:AB$ DealDamage | Cost$ 3 Unattach<Equipment.Attached/Equipment attached to NICKNAME> | PrecostDesc$ Throw . . . — | ValidTgts$ Any | TgtPrompt$ Select any target to distribute damage to | NumDmg$ X | TargetMin$ 1 | TargetMax$ 3 | DividedAsYouChoose$ X | SpellDescription$ He deals damage equal to that Equipment's mana value divided as you choose among one, two, or three targets.
```

Comment:
@Hanmac
any idea how we can support this?

Reply:
What exactly is the problem?

Reply:
I needed partial payment decision early for the division
but should work now